CHANGELOG.md

```diff
@@ -1,66 +1,106 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.116.0] - 2023-03-28
+
+### Added
+
+- Smart Transfer Protocol: implemented Pull flow based on web-sockets
```

Comment:
Would be nice to give a short example, e.g. "if you run `kamu system api-server` you can pull from it as ..."

Reply:
Added